### ISSUE TITLE
npm install and npm start return error

### ISSUE THREAD

**Issue by elliotsyoung** [bradtraversy/redux_crash_course#1]
I'm getting an error where the app won't run after I do `npm install` and `npm start`, I've attached an image of the page in my browser:

<img width="1792" alt="screen shot 2018-05-09 at 8 32 01 pm" src="https://user-images.githubusercontent.com/16292462/39850786-21add596-53c8-11e8-870b-0f2de46299cd.png">

**Comment by monzoor** [comment]
Having same error.

**Comment by velmuruganui** [comment]
Yes having same error... Chrome working fine another browser not working.

**Comment by Vins374** [comment]
You have to install redux dev tool chrome extension and react chrome extension.

**Comment by maisonm** [comment]
Installing both Chrome Redux and React extensions fixes this error.
